src/redux/api/api.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] Please do not disable your `eslint` linters check rather fix the error and allow it to pass successfully(:heavy_check_mark: )